### DISCUSSION TITLE
Trouble binding row data to a <FluentAnchor Href> attribute within a <TemplateColumn> of a <FluentDataGrid>

### DISCUSSION THREAD

**Discussion by GeoffreyB** [microsoft/fluentui-blazor#1884] [Q&A] [answered]
I am attempting to create an Admin page which enables CRUD operations on a list of users. The model is populated by a service that retrieves an IQueryable. The PropertyColumn Property is being populated correctly using 
`Property="@(x +> x.Email)"` 
However I am having issues concatenating the string for the Href attribute of the FluentAnchor i.e. adding the user Id parameter to the link. The following code does not create a compiler error:
`<FluentDataGrid Items="@users" ResizableColumns=true Pagination="@pagination" GridTemplateColumns="1fr 1fr 1fr 1fr 0.7fr 0.7fr 0.7fr" Style="height: 405px;overflow:auto;">
  <PropertyColumn Property="@(x => x.LastName)" Sortable="true" Tooltip="true" Title="Last Name" Align=Align.Start />
  <PropertyColumn Property="@(x => x.FirstName)" Sortable="true" Tooltip="true" Title="First Name" Align=Align.Start />
  <PropertyColumn Property="@(x => x.Email)" Sortable="true" Tooltip="true" Title="Email Address" Align=Align.Start />
  <PropertyColumn Property="@(x => x.PhoneNumber)" Sortable="true" Tooltip="true" Title="Phone" Align=Align.End />
  <TemplateColumn Tooltip="true" TooltipText="@(x => "Edit " +  x.FirstName + " " + x.LastName)" Title="" Align="Align.Center">
    <FluentAnchor Href="/admin/users/edituser/" + @(x => x.Id)  IconStart="@(new Icons.Regular.Size20.DocumentEdit())" Style="color:blue">Edit</FluentAnchor>
  </TemplateColumn>
</FluentDataGrid>`
However the following screenshot is what is displayed:
![image](https://github.com/microsoft/fluentui-blazor/assets/1011077/dbeda7ea-233b-4804-ad22-74f892e64cea)

**What is the correct syntax to create the string to populate the Href attribute?**

**Comment by vnbaaij** [comment, marked as answer]
Hi,

This is not really a library thing but more a general Blazor question, I think.

The FluentAnchor inside the templatecolumn does not know about 'x'. You need to supply it that knowledge by using the @context attribute
